_posts/2017/2017-04-06-javaland-2017-in-phantasialand.md

```diff
@@ -0,0 +1,131 @@
+---
+layout: post
+title: "JavaLand 2017 - Welcome to Phantasialand"
+date: "2017-04-06 15:16:17"
+image: blog-header/microxchg-2017.jpg
+categories: events
+authors: ["Benjamin, Bastian"]
+---
+
+The [JavaLand 2017](https://www.javaland.eu/en/javaland-2017/) conference took place for the 4th time in the famous [Phantasialand](http://www.phantasialand.de/en/) holiday park in Brühl, near Cologne. The program consisted of [talks](https://programm.javaland.eu/2017/schedule.html) on the frist two days and [in-depth workshops](https://www.javaland.eu/de/programm/schulungstag) at the last and third day.
```

Comment:
please use one sentence per line.
"The 4th Javaland conference took place in the..."
"...on the **first** two days.."